streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -98,8 +97,9 @@
 
     private StreamPartitionAssignor partitionAssignor = null;
 
-    private long lastClean;
-    private long lastCommit;
+    private long currTimeMs;
```

Comment:
I'm trying to save one `time.milliseconds()` each time when we want to measure latency, since it is triggered multiple times within one iteration of the thread. The basic idea is that the end time of a previous phase can be just used as the start time of the next phase, and therefore the name `currTimeMs`.